parser/parser_test.go

```diff
@@ -331,7 +331,19 @@ var parseTests = []parseTest{
 	{
 		"SimpleService",
 		"service TestService {\n  rpc Foo(In) returns (Out);\n}\n message In{} message Out{}",
-		`service { name: "TestService" method { name:"Foo" input_type:".In" output_type:".Out" } }` +
+		`service { name: "TestService" method { name:"Foo" input_type:".In" output_type:".Out" client_streaming: false server_streaming: false } }` +
+			`message_type:{name:"In"} message_type:{name:"Out"}`,
+	},
+	{
+		"SimpleServiceWithMessageCalledStream",
```

Comment:
Ah, I missed that. Right. Okay, keep this test. You're right that MethodsAndStreams looks pointless; I should have looked more closely.